Coffee3.4/src/CoffeeBrush.java

```diff
@@ -0,0 +1,206 @@
+        System.out.println("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino:");
+        int num = scan.nextInt();
+
+        /**
+         * espresso stuff
+         * */
+        if (num == espresso) {
+            if (water >= e_water && coffee >= e_coffee) {
+                System.out.println("I have enough resources, making you a coffee!");
+                water = water - e_water;
+                coffee = coffee - e_coffee;
+                --cups;
+                money = money + e_cost;
+            } else if (water < e_water) {
+                System.out.println("Sorry, not enough water!");
+            } else {
+                System.out.println("Sorry, not enough coffee!");
+            }
+        }
+
+        /**
+         * latte stuff
+         * */
+         else if (num == latte) {
+            if (water >= l_water && milk >= l_milk && coffee >= l_coffee) {
+                System.out.println("I have enough resources, making you a coffee!");
+                water = water - l_water;
+                milk = milk - l_milk;
+                coffee = coffee - l_coffee;
+                --cups;
+                money = money + l_cost;
+            } else if (water < l_water) {
+                System.out.println("Sorry, not enough water!");
+            } else if (milk < l_milk) {
+                System.out.println("Sorry, not enough milk!");
+            } else {
+                System.out.println("Sorry, not enough coffee!");
+            }
+
+        }
+
+        /**
+         * cappuccino stuff
+         * */
+         else if (num == cappuccino) {
+            if (water >= c_water && milk >= c_milk && coffee >= c_coffee) {
+                System.out.println("I have enough resources, making you a coffee!");
+                water = water - c_water;
+                milk = milk - c_milk;
+                coffee = coffee - c_coffee;
+                --cups;
+                money = money + c_cost;
+            } else if (water < c_water) {
+                System.out.println("Sorry, not enough water!");
+            } else if (milk < c_milk) {
+                System.out.println("Sorry, not enough milk!");
+            } else {
+                System.out.println("Sorry, not enough coffee!");
+            }
+        }
+        else {
+            System.out.println("Invalid action!");
+        }
+    }
+
+    /**
+     * remaining method
+     * */
+    private void remaining(){
+        System.out.println("The coffee machine has:");
+        System.out.println(water + " of water");
+        System.out.println(milk + " of milk");
+        System.out.println(coffee + " of coffee beans");
+        System.out.println(cups + " of disposable cups");
+        System.out.println(money + "$ of money");
+    }
+
+    /**
+     * take method
+     * */
+    private void take() {
+        System.out.println("I gave you $" + money);
+        money = money - money;
+    }
+
+    /**
+     * fill method
```

Comment:
@aniterstepanyan try to write more meaning comments note like a "this is my code" or just method name

Reply:
Done